core/EE_Deprecated.core.php

```diff
@@ -1391,19 +1393,35 @@ public static function is_update_available($basename = '')
 );
 /**
  * Filters the CSV row to make it appear like the old labels (which were "$pretty_name[$field_name]").
+ * @deprecated since $VID:$. This only exists for backward compatibility with code snippets.
```

Comment:
This should be:

```
* @deprecated $VID:$ ...description
```
I realize that you may have gone by the deprecated tag on the `ee_deprecated_update_attendee_registration_form_old` but that is also wrong there.